carfield.mk

```
@@ -45,20 +45,20 @@ include $(CAR_ROOT)/bender-safed.mk
 # Nonfree components #
 ######################
 
-CAR_NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:carfield/carfield-nonfree.git
-CAR_NONFREE_COMMIT ?= 54ce7e49
+CAR_NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:astral/astral-nonfree.git
+CAR_NONFREE_COMMIT ?= 711d0097
 
 ## @section Carfield platform nonfree components
 ## Clone the non-free verification IP for Carfield. Some components such as CI scripts and ASIC
 ## implementations with tech-specific resources are not open-sourced and contained in a `nonfree`
 ## folder cloned from a remote location, whose access is restricted. If you do not have access, this
 ## step will be skipped and the usage of the repository will **not** be compromised.
 car-nonfree-init:
-	git clone $(CAR_NONFREE_REMOTE) $(CAR_ROOT)/nonfree
-	cd nonfree && git checkout $(CAR_NONFREE_COMMIT)
-	cd nonfree/intel16 && icdesign intel16 -update all -nogui
+	git clone $(CAR_NONFREE_REMOTE) nonfree
+	cd $(CAR_ROOT)/nonfree && git checkout $(CAR_NONFREE_COMMIT)
+	#cd nonfree/intel16 && icdesign intel16 -update all -nogui
```

I suggest to be fully consistent with the directories to avoid any accidents.

```suggestion
	git clone $(CAR_NONFREE_REMOTE) $(CAR_ROOT)/nonfree
	cd $(CAR_ROOT)/nonfree && git checkout $(CAR_NONFREE_COMMIT)
	#cd nonfree/intel16 && icdesign intel16 -update all -nogui
```

---

Good spot, thanks!